### PR TITLE
Rate Limiting notifications on read/write endpoints

### LINKED ISSUE #13422: The UI should display easy to understand notifications on rate/limiting errors for queries
As a user makes a query or series of queries that exceed usage rates- the backend will reject the request with an HTTP 429 [#3109](https://github.com/influxdata/idpe/issues/3109)

The UI should display a notification that is easy for the user to understand that a rate limit has been reached.

In particular, let's test a dashboard in expectation that it may be easy for users to exceed max: 2 concurrency limits on queries. 

Check: data explorer and dashboard View Editor Overlays

### PR DESCRIPTION
**Pull request by ebb-tide**
Closes #13422

### CODE REVIEW

File: ui/src/shared/copy/notifications.ts [hunk 1]
```diff
@@ -948,3 +948,42 @@ export const deleteBucketFailed = (): Notification => ({
   ...defaultErrorNotification,
   message: 'Failed to delete bucket',
 })
+
+// Limits
+export const writeLimitReached = (errorMessage: string): Notification => ({
+  ...defaultErrorNotification,
+  message: `You have reached a write limit on this resource: ${errorMessage}.`,
+  duration: INFINITE,
+})
```
Comment (lukevmorris) [inline ui/src/shared/copy/notifications.ts]: I wonder if `writeLimitReached` and `readLimitReached` should become convenience mehtods for `readWriteLimitReached`?

File: ui/src/dataLoaders/actions/dataLoaders.ts [hunk 2]
```diff
@@ -547,6 +549,13 @@ export const writeLineProtocolAction = (
     dispatch(setLPStatus(RemoteDataState.Done))
   } catch (error) {
     dispatch(setLPStatus(RemoteDataState.Error))
+
+    const errorMessage = _.get(error, 'response.data.message', '')
+    console.error(errorMessage || error)
+
+    if (error.response.status == RATE_LIMIT_ERROR_STATUS) {
```
Comment (lukevmorris) [inline ui/src/dataLoaders/actions/dataLoaders.ts]: I can has threequals?

File: ui/src/shared/copy/notifications.ts (hunk 1 above)
Comment (ebb-tide) [reply ui/src/shared/copy/notifications.ts]: I was using the writeLimitReached for line protocol writes, so it felt reasonable here. I'm expecting feedback on error messages.

File: ui/src/dataLoaders/actions/dataLoaders.ts (hunk 2 above)
Comment (ebb-tide) [reply ui/src/dataLoaders/actions/dataLoaders.ts]: yes you most certainly can

File: ui/src/limits/actions/index.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+) => {
+  try {
+    const {
+      orgs: {org},
+    } = getState()
+
+    const limits = await getReadWriteLimitsAJAX(org.id)
+
+    const readLimit = limits.read.status === LimitStatus.EXCEEDED
+    const writeLimit = limits.write.status === LimitStatus.EXCEEDED
```
Comment (ischolten) [inline ui/src/limits/actions/index.ts]: nit: `readLimit` and `writeLimit` sound like they would be the actual limit like a number. seems like these should be something along the lines of `readLimitReached`

File: ui/src/shared/copy/notifications.ts [hunk 4]
```diff
@@ -519,7 +519,7 @@ export const dashboardExportFailed = (
 
 export const dashboardCreateFailed = () => ({
   ...defaultErrorNotification,
-  message: 'Failed to created dashboard.',
+  message: 'Failed to create dashboard.',
```
Comment (ischolten) [inline ui/src/shared/copy/notifications.ts]: nice catch!

File: ui/src/shared/copy/notifications.ts (hunk 1 above)
Comment (ischolten) [reply ui/src/shared/copy/notifications.ts]: +1

Comment (lukevmorris) [reply ui/src/shared/copy/notifications.ts]: I really like the message you've written on L974. Seems like a good wording until we introduce response headers that will return more specific info.

File: ui/src/limits/actions/index.ts (hunk 3 above)
Comment (ebb-tide) [reply ui/src/limits/actions/index.ts]: 👍

**Pull request merged**